lib/printer.ts

```diff
@@ -602,7 +602,14 @@ function genericPrintNoParens(path: any, options: any, print: any) {
         parts.push(" from ");
       }
 
-      parts.push(path.call(print, "source"), ";");
+      parts.push(path.call(print, "source"));
+
+      if (n.assertions && n.assertions.length > 0) {
+          const [assert] = n.assertions;
```

Comment:
I see from the proposal that there's only one key they expect to be in use for now, `type`. But is there any reason not to print however many assertions as there are in the AST?

Reply:
There is no reason no to print as many assertions as there are in the AST. Just added.